doc/whats_new/v0.20.rst

```diff
@@ -1028,18 +1028,26 @@ Miscellaneous
   :func:`metrics.pairwise_distances_chunked`. See :ref:`working_memory`.
   :issue:`10280` by `Joel Nothman`_ and :user:`Aman Dalmia <dalmia>`.
 
+- |Feature| The version of :mod:`joblib` bundled with Scikit-learn is now 0.12.
+  This uses a new default multiprocessing implementation, named `loky
+  <https://github.com/tomMoral/loky>`_. While this may incur some memory and
+  communication overhead, it should provide greater cross-platform stability
+  than relying on Python standard library multiprocessing. :issue:`11741` by
+  the Joblib developers, especially :user:`Thomas Moreau <tomMoral>` and
```

Comment:
The joblib changelog is,
```
Implement the 'loky' backend [..]. This backend relies on a robust implementation of concurrent.futures.ProcessPoolExecutor with spawned processes that can be reused accross the Parallel calls. This fixes the bad interation with third paty libraries relying on thread pools, described in pythonhosted.org/joblib/parallel.html#bad-interaction-of-multiprocessing-and-third-party-libraries
```
I couldn't find anything about "memory and communication overhead".

Maybe rather adapt that changelong, unless we get some feedback from joblib devs (cc @lesteve , @tomMoral ) ?

Reply:
Not an expert here but my LGTM is based on the doc below:
https://joblib.readthedocs.io/en/latest/parallel.html#thread-based-parallelism-vs-process-based-parallelism
By default joblib.Parallel uses the 'loky' backend module to start separate Python worker processes to execute tasks concurrently on separate CPUs. This is a reasonable default for generic Python programs but can induce a significant overhead as the input and output data need to be serialized in a queue for communication with the worker processes.

Reply:
Thanks, makes sense then.